### PR TITLE
Make otel error log less scary

### PR DESCRIPTION
**Pull request by 0sewa0**
## Description

Setting up OTel requires a specific secret to be created.
This is totally optional, so most of the time this secret won't be there, so logging the error is a bit excessive.

## How can this be tested?

Deploy the operator without having the Otel secret setup,and check the log of the operator is less scary

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]
thank you!! was in my list!

File: pkg/util/dtotel/otel.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ type shutdownFn func(ctx context.Context) error
 func Start(ctx context.Context, otelServiceName string, apiReader client.Reader, webhookNamespace string) func() {
 	endpoint, apiToken, err := getOtelConfig(ctx, apiReader, webhookNamespace)
 	if err != nil {
-		log.Info("failed to read OpenTelementry config secret", "err", err.Error())
+		log.Info("Otel was not configured, add the secret if you want to enable this functionality.")
```
Comment (chrismuellner) [inline pkg/util/dtotel/otel.go:25]:
```suggestion
		log.Info("Opentelemetry instrumentation was not configured")
```

File: pkg/util/dtotel/otel.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ type shutdownFn func(ctx context.Context) error
...
 
 		return setupNoopOTel()
 	}
```
Comment (chrismuellner) [inline pkg/util/dtotel/otel.go:28]: should we have a log as well that indicates that otel was enabled?

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: pkg/util/dtotel/otel.go (hunk 2 above)
Comment (0sewa0) [reply pkg/util/dtotel/otel.go:28]: IMO not necessary

**Review by chrismuellner — APPROVED** [review]

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**